pkg/ingester/ingester.go

```diff
@@ -160,6 +160,10 @@ type Config struct {
 	InstanceLimitsFn func() *InstanceLimits `yaml:"-"`
 
 	IgnoreSeriesLimitForMetricNames string `yaml:"ignore_series_limit_for_metric_names" category:"advanced"`
+
+	utilizationLimitsEnabled       bool    `yaml:"-"`
```

Comment:
[nit] Remove this. You don't need it (see below).

Reply:
Done.